en/pl/jsp.rst

```diff
@@ -137,30 +130,46 @@ You can refer to the :ref:`jsp-load-java`.
 
     % loadjava demodb SpCubrid.class
 
+Registering Stored Functions/Procedures
+=======================================
 
-Publish the loaded Java class
------------------------------
+CUBRID requires a process to register (publish) Java classes so that Java methods can be called from SQL statements or Java applications.
+Create a CUBRID stored function as follows to register the Java class.
 
-In CUBRID, it is required to publish Java classes to call Java methods from SQL statements or Java applications.
-Create a CUBRID stored function and publish the Java class as shown below.
-For more details, see :ref:`call-specification`.
+::
+
+    ...
+    AS LANGUAGE JAVA NAME '<java_call_specification>';
+
+In the string following JAVA NAME, specify the class and method names to call the method of the Java class. This is called a **Java Call Specification**.
 
 .. code-block:: sql
 
     CREATE FUNCTION hello() RETURN STRING 
     AS LANGUAGE JAVA 
     NAME 'SpCubrid.HelloCubrid() return java.lang.String';
 
-.. CREATE OR REPLACE FUNCTION is allowed from 10.0: CUBRIDSUS-6542
+.. _call-specification:
+
+Java Call Specification
+-----------------------------
 
-Or with **OR REPLACE** syntax, you can replace the current stored function/procedure or create the new stored function/procedure.
+You must publish Java classes by using call specifications because it is not known how a function in a class will be called by SQL statements or Java applications when Java classes are loaded.
```

Comment:
```suggestion
You must publish Java classes by using call specifications because it is not known how a function in a class will be called by SQL statements or applications when Java classes are loaded.
```

Reply:
한글 매뉴얼에 반영해주세요.